code/Common/Importer.h

```diff
@@ -129,14 +129,8 @@ ImporterPimpl::ImporterPimpl() AI_NO_EXCEPT
 , mIsDefaultHandler( false )
 , mProgressHandler( nullptr )
 , mIsDefaultProgressHandler( false )
-, mImporter()
 , mPostProcessingSteps()
```

Comment:
Why?

Reply:
because the constructor for those fields it will be called anyway